src/main/webapp/app/exam/liveExamExerciseUpdateService.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class LiveExamExerciseUpdateService {
+    private examExerciseIdAndProblemStatementSource = new BehaviorSubject({ exerciseId: -1, problemStatement: '' });
```

Comment:
I think it would be good to add an interface here, that defines the type of objects managed by this BehaviorSubject.
Please consider adding something like: 
```typescript
interface LiveExamExerciseUpdate {
  exerciseId?: number;
  problemStatement?: string;
}
// ...
... new BehaviorSubject<LiveExamExerciseUpdate>({...});
```
Adding this typing information will make life easier for people who have to work with your API.

Reply:
Thank you for your advice, I added the interface and changed the related code in the exam navigation bar.

@lschlesinger are my changes correct? I have not used interfaces in typescript before.
I assume you should use interfaces in ts when you want to type check multiple values (like an object) and not just one (that's why the other service where only a number is exchanged should not receive an interface, correct?)

Reply:
Yes, interfaces define the structure of an object. Your changes are correct!